RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+*bug*: handling edge cases where image is absolute, does not exist, or cannot be identified by Pillow.
```

Comment:
where image *location is non-local to the Pelican site source*,

Reply:
^^ I realize this is rather wordy, but *absolute* just sounds off to me.

Reply:
Ok, fixed!